### ISSUE TITLE
Failed to acquire RNG protocol: Not Found

### ISSUE THREAD

**Issue by mudrii** [NixOS/nixpkgs#72773]
**Describe the bug**
During the boot getting message
SHA256 validated
**Failed to acquire RNG protocol: Not Found**

**To Reproduce**
1. In configure.nix using "kernelPackages = pkgs.linuxPackages_latest;"
2.  Message during the boot **Failed to acquire RNG protocol: Not Found**

**Expected behaviour**
No error message on RNG

**Additional context**
NixOS running on older Desktop PC 
Specs:
Intel(R) i7-4770K CPU
ASUS MAXIMUS VI GENE

```
bootctl
System:
     Firmware: UEFI 2.31 (American Megatrends 4.654)
  Secure Boot: disabled
   Setup Mode: user

Current Boot Loader:
      Product: systemd-boot 243
     Features: ✓ Boot counting
               ✓ Menu timeout control
               ✓ One-shot menu timeout control
               ✓ Default entry control
               ✓ One-shot entry control
               ✓ Support for XBOOTLDR partition
               ✓ Support for passing random seed to OS
          ESP: /dev/disk/by-partuuid/11c984ea-e7a4-4eb0-a235-476c97f80cfe
         File: └─/EFI/systemd/systemd-bootx64.efi

Random Seed:
 Passed to OS: yes
 System Token: set
       Exists: yes

...
...
...

Default Boot Loader Entry:
        title: NixOS (Generation 56 NixOS 19.09.1019.c5aabb0d603, Linux Kernel 5.3.8, Built on 2019-11-04)
           id: nixos-generation-56
       source: /boot/loader/entries/nixos-generation-56.conf
      version: Generation 56 NixOS 19.09.1019.c5aabb0d603, Linux Kernel 5.3.8, Built on 2019-11-04
   machine-id: 56c6d4e336564850a9873710478633b1
        linux: /efi/nixos/pqwx8b5imy39vchh925hqh6zclw7a2pv-linux-5.3.8-bzImage.efi
       initrd: /efi/nixos/jf1c9z9m21p6cqbr0pbsnc599kax55kp-initrd-linux-5.3.8-initrd.efi
      options: systemConfig=/nix/store/r535x985xpp2b8xl8xrh0f14b4hpv9km-nixos-system-desktop-nixos-19.09.1019.c5aabb0d603 init=/nix/store/r535x985xpp2b8xl8xrh0f14b4hpv9km-nixos-system-desktop-nixos-19.09.1019.c>

```

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.3.8, NixOS, 19.09.1019.c5aabb0d603 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"nixos-19.09.1019.c5aabb0d603, home-manager, unstable-20.03pre196586.b943338ea58"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Note: On arch linux forum is a similar bug posted > https://bugs.archlinux.org/task/63698
Bug Opened on systemd > https://github.com/systemd/systemd/issues/13503

**Comment by scalavision** [comment]
I also got this. I have the same cpu.

**Comment by trusktr** [comment]
I have the same issue. In case it helps, here's my details:

```
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.93, NixOS, 19.09.1778.db3e8325a9b (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"trusktr, nixos-19.09.1778.db3e8325a9b"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

and

```
# nix run nixpkgs.pciutils -c lspci
00:00.0 Host bridge: Intel Corporation Xeon E3-1200 v5/E3-1500 v5/6th Gen Core Processor Host Bridge/DRAM Registers (rev 08)
00:02.0 VGA compatible controller: Intel Corporation Skylake GT2 [HD Graphics 520] (rev 07)
00:04.0 Signal processing controller: Intel Corporation Xeon E3-1200 v5/E3-1500 v5/6th Gen Core Processor Thermal Subsystem (rev 08)
00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21)
00:14.2 Signal processing controller: Intel Corporation Sunrise Point-LP Thermal subsystem (rev 21)
00:16.0 Communication controller: Intel Corporation Sunrise Point-LP CSME HECI #1 (rev 21)
00:17.0 RAID bus controller: Intel Corporation 82801 Mobile SATA Controller [RAID mode] (rev 21)
00:1c.0 PCI bridge: Intel Corporation Sunrise Point-LP PCI Express Root Port #5 (rev f1)
00:1d.0 PCI bridge: Intel Corporation Device 9d1a (rev f1)
00:1f.0 ISA bridge: Intel Corporation Sunrise Point-LP LPC Controller (rev 21)
00:1f.2 Memory controller: Intel Corporation Sunrise Point-LP PMC (rev 21)
00:1f.3 Audio device: Intel Corporation Sunrise Point-LP HD Audio (rev 21)
00:1f.4 SMBus: Intel Corporation Sunrise Point-LP SMBus (rev 21)
00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
01:00.0 Network controller: Intel Corporation Wireless 8260 (rev 3a)
02:00.0 Unassigned class [ff00]: Realtek Semiconductor Co., Ltd. RTS525A PCI Express Card Reader (rev 01)
```

**Comment by mudrii** [comment]
Seems issue was fixed on systemd
https://github.com/systemd/systemd/pull/13510#event-2758825938

Hope we can get an update in nixos soon

**Comment by NilsIrl** [comment]
I have this on:

* `i7 6700k`
* `Asus z170 pro graming`

**Comment by Emantor** [comment]
This should be fixed on unstable by #85334.

**Comment by 101100** [comment]
Any chance the fix will make it into 20.03?

**Comment by Emantor** [comment]
> Any chance the fix will make it into 20.03?

Currently the fix has no backport PR for 20.03. Given that 20.03 strives for stability, I suspect that a backport could introduce unwanted breakage on 20.03. Keep in  mind that this is a purely cosmetic issue and IMO does not warrant a backport. On the other hand, feel free to create a backport PR and discuss it there.
